wallet/wallet.go

```diff
@@ -3871,17 +3871,8 @@ func (w *Wallet) ImportVotingAccount(ctx context.Context, xpriv *hdkeychain.Exte
 		return 0, errors.E(op, err)
 	}
 
-	if n, err := w.NetworkBackend(); err == nil {
-		// Internal addresses are never used from the voting account.
-		extAddrs, err := deriveChildAddresses(extKey, 0, w.gapLimit, w.chainParams)
-		if err != nil {
-			return 0, errors.E(op, err)
-		}
-		err = n.LoadTxFilter(ctx, false, extAddrs, nil)
-		if err != nil {
-			return 0, errors.E(op, err)
-		}
-	}
+	// TODO: Increment external and internal indexes for known used stake
+	// submission scripts.
```

Comment:
I'm not sure if this is adequate. Will look into it and maybe tack on another commit to solve the TODO.